Majasdarbs/03-majasdarbs/cv-css/css/responsivness.css

```diff
@@ -0,0 +1,30 @@
+@media only screen and (max-width: 700px){
```

Comment:
Check common breakpoints, might be useful https://www.freecodecamp.org/news/css-media-queries-breakpoints-media-types-standard-resolutions-and-more/